tests/vm_migration_test.go

```diff
@@ -168,6 +168,59 @@ var _ = Describe("VmMigration", func() {
 			}, TIMEOUT, POLLING_INTERVAL).Should(Equal(1))
 			close(done)
 		}, 60)
+		It("Should migrate the VM with a non-default namespace", func(done Done) {
```

Comment:
We could make the other migration test parametrized and give it different namespaces. https://onsi.github.io/ginkgo/#table-driven-tests